### ISSUE TITLE
importing module nupicvision fails

### ISSUE THREAD

**Issue by breznak** [numenta/nupic.vision#8]
After https://github.com/numenta/nupic.core/pull/360 
which allows to `export NUPIC_REGIONS_PATH=/abs/path/to/nupic.vision/regions/` and use "3rd-party regions" in nupic, I still have failures running the test: 
https://github.com/nupic-community/nupic.vision/blob/master/tests/unit/nupicvision/regions/image_sensor_test.py

with: 
```
(pyenv)mmm@mmm-U2442:~/devel/nupic/projects/nupic-community/nupic.vision$ python tests/unit/nupicvision/regions/image_sensor_test.py 
tests/unit/nupicvision/regions/image_sensor_test.py:130: SyntaxWarning: import * only allowed at module level
  def testRunPCANode(self):
Traceback (most recent call last):
  File "tests/unit/nupicvision/regions/image_sensor_test.py", line 31, in <module>
    from nupicvision.regions.ImageSensor import ImageSensor
ImportError: No module named nupicvision.regions.ImageSensor
```

while manually, it seems to work (?!):
```
(pyenv)mmm@mmm-U2442:~/devel/nupic/projects/nupic-community/nupic.vision$ python
Python 2.7.6 (default, Mar 22 2014, 22:59:56) 
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from nupicvision.regions.ImageSensor import ImageSensor
>>> 
```

@scottpurdy @chetan51 Any idea? I think I must just wrongly set `__init__.py` or something...?

**Comment by chetan51** [comment]
Do you have `nupic.vision` properly installed? Can you import it in a Python shell started from your home directory?

**Comment by breznak** [comment]
> Do you have nupic.vision properly installed? Can you import it in a Python shell started from your home directory? 

@chetan51 no, that would be the problem! But do I have to "install" for local use? From the `nupic.vision/` dir the `python tests/...` fails but starting python and `import nupicvision` works fine. 

@david-ragazzi could you help me with a simple `setup.py` please?

**Comment by chetan51** [comment]
@breznak When you start python and `import nupicvision` from the `nupic.vision` directory, it imports it using relative reference. If you want to be able to import it elsewhere (for example your home directory or from tests), you'll need to install it using the `python setup.py install` command.

**Comment by scottpurdy** [comment]
Supposedly resolved by numenta/nupic.core#359

**Comment by breznak** [comment]
@scottpurdy should this be closed? As long as nobody provided the `setup.py` it would not be resolved, as Chetan pointed out: 
> If you want to be able to import it elsewhere (for example your home directory or from tests), you'll need to install it using the python setup.py install command.

**Comment by scottpurdy** [comment]
Sure we can keep it open until there is a setup.py in place

**Comment by scottpurdy** [comment]
This should be resolved as of #32
